### PR TITLE
add "add badge to your project" section

### PR DESCRIPTION
**Pull request by jbmoelker**
## Summary

Proposed changes:
- add "add badge to your project" section. incl. summary, why and how with copy-paste code snippets.
## Checklist
- [X] The changes only affect or contain one style guide rule or section.
- [X] The changes are in [style guide format](/CONTRIBUTING.md#format).
- [X] The new style guide section has an entry in the [Table of Contents](/README.md#table-of-contents) (only if changes introduce new section).
- [X] The changes can be merged into the target branch without conflicts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -718,3 +721,34 @@ Example demo file in `city-tag` module:
...
+
+## Add badge to your project
+
+You can use the *RiotJS Style Guide badge* to link to this guide:
+
+[![RiotJS Style Guide badge](https://cdn.rawgit.com/voorhoede/riotjs-style-guide/master/riotjs-style-guide.svg)](https://github.com/voorhoede/riotjs-style-guide)
+
+### Why?
+
+Inform other developers your project is following the RiotJS Style Guide. And let them know where they can find this guide.
```
Comment (phortuin) [inline README.md:734]: Why not separate points?

File: README.md [hunk 2]
```diff
@@ -718,3 +721,34 @@ Example demo file in `city-tag` module:
...
+```markdown
+[![RiotJS Style Guide badge](https://cdn.rawgit.com/voorhoede/riotjs-style-guide/master/riotjs-style-guide.svg)](https://github.com/voorhoede/riotjs-style-guide)
+```
+
+Or html:
+
+```html
+<a href="https://github.com/voorhoede/riotjs-style-guide">
+    <img alt="RiotJS Style Guide badge"
+    	 src="https://cdn.rawgit.com/voorhoede/riotjs-style-guide/master/riotjs-style-guide.svg">
```
Comment (phortuin) [inline README.md:749]: `//cdn.rawgit.com` instead of `https://`?

**Comment by jbmoelker** [comment]
@phortuin relative protocols have become an anti-pattern:

> Now that SSL is encouraged for everyone and doesn’t have performance concerns, this technique is now an anti-pattern. If the asset you need is available on SSL, then always use the https:// asset.
> -- [Paul Irish](http://www.paulirish.com/2010/the-protocol-relative-url/)

**Comment by jbmoelker** [comment]
@phortuin in response to https://github.com/voorhoede/riotjs-style-guide/pull/41#discussion_r57727716:

Considered as separate points. Felt like a bit much. Prefer bullet list? I'm happy to change it.

**Comment by phortuin** [comment]
@jbmoelker in response to #41, leave it, making it bullets would suggest that they are actual "pros" instead of just our kind request to use our stuff

**Pull request merged**